source/backend/apimodels/Models/Concepts/Property/PropertyMap.cs

```diff
@@ -64,17 +64,55 @@ public void Register(TypeAdapterConfig config)
                 .Inherits<Entity.IBaseEntity, BaseConcurrentModel>();
 
             config.NewConfig<PropertyModel, Entity.PimsProperty>()
-                .Map(dest => dest.Internal_Id, src => src.Id)
+            .Map(dest => dest.Internal_Id, src => src.Id)
+            .Map(dest => dest.Pid, src => src.Pid)
+            .Map(dest => dest.Pin, src => src.Pin)
+            .Map(dest => dest.SurveyPlanNumber, src => src.PlanNumber)
+            .Map(dest => dest.PropertyStatusTypeCode, src => src.Status.Id)
+            .Map(dest => dest.PropertyTypeCode, src => src.PropertyType.Id)
+            .Map(dest => dest.Address, src => src.Address)
+            .Map(dest => dest.DistrictCode, src => src.District.Id)
+            .Map(dest => dest.RegionCode, src => src.Region.Id)
+            .Map(dest => dest.Location, src => src.Location)
+            .Map(dest => dest.Boundary, src => src.Boundary)
+            .Map(dest => dest.GeneralLocation, src => src.GeneralLocation)
+
+            .Map(dest => dest.IsRetired, src => src.IsRetired)
+            .Map(dest => dest.IsRwyBeltDomPatent, src => src.IsRwyBeltDomPatent)
+            .Map(dest => dest.PphStatusTypeCode, src => src.PphStatusTypeCode)
+
+            .Map(dest => dest.Notes, src => src.Notes)
+            .Map(dest => dest.IsOwned, src => src.IsOwned)
+
+            // multi-selects
+            .Map(dest => dest.PimsPropPropAnomalyTyps, src => src.Anomalies)
+            .Map(dest => dest.PimsPropPropTenureTyps, src => src.Tenures)
+            .Map(dest => dest.PimsPropPropRoadTyps, src => src.RoadTypes)
+
+            .Map(dest => dest.LandArea, src => src.LandArea)
+            .Map(dest => dest.PropertyAreaUnitTypeCode, src => src.AreaUnit.Id)
```

Comment:
unlike the other dereferences on this mapper, this one appears to be nullable which may cause an NPE?

Reply:
This is the same code that was there before (see old line 97). The mapping between The model and the entity should be the same as before; I guess the diff is showing because of indentation